pkg/trace/stats/concentrator_test.go

```diff
@@ -381,42 +430,121 @@ func TestConcentratorSublayersStatsCounts(t *testing.T) {
 
 	// Start with the first/older bucket
 	receivedCounts = stats[0].Counts
-	expectedCountValByKey := map[string]int64{
-		"query|_sublayers.duration.by_service|env:none,resource:resource1,service:A1,sublayer_service:A1": 2000,
+	expectedCountValByKey := map[string]float64{
+		"query|_sublayers.duration.by_service|env:none,resource:resource1,service:A1,sublayer_service:A1": 2200,
```

Comment:
Needed to update sublayer values in accordance with the added spans (7 and 10 above).

Reply:
You should have an _additional_ test to validate your change rather than having to update all these values (see comment above)

Reply:
Agree with Pete here. This is getting messy.